scripts/apps/ingest/views/settings/ingest-sources-content.html

```diff
@@ -109,7 +109,7 @@ <h3 class="modal__heading" translate ng-hide="provider._id" translate>Add New So
                             <a  href=""
                                 ng-repeat="fileType in fileTypes"
                                 title="{{fileType.type}}"
-                                ng-click="addOrRemoveFileType(fileType.type)"
+                                ng-click="addOrRemoveFileType(fileType.type); editForm.$setDirty()"
```

Comment:
Set it dirty in the function itself instead of where it's called so if it's called in another place, you don't need to remember to set dirty as well.

Reply:
fixed it.